csharp-architectural-patterns/ClientServerArchitecture/ClientServerArchitecture/Program.cs

```diff
@@ -0,0 +1,45 @@
+using ClientServerArchitecture.Data;
+using ClientServerArchitecture.Data_Access;
+using ClientServerArchitecture.Services;
+using Microsoft.EntityFrameworkCore;
+
+namespace ClientServerArchitecture
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
```

Comment:
Since we are in .NET 8, let's use top-level statements instead of having the `Program` class and the `Main()` method shown.